src/components/Date/Date.unit.spec.jsx

```diff
@@ -204,8 +189,38 @@ describe('<Date>', () => {
     date.month.dirty = true;
     date.day.dirty = true;
 
+    return axeCheck(<Date date={date} onValueChange={() => {}} />);
+  });
+
+  it('renders aria-describedby on elements', () => {
+    const date = {
+      day: makeField(1),
+      month: makeField(12),
+      year: makeField(2010),
+    };
+    const tree = shallow(
+      <Date date={date} onValueChange={() => {}} ariaDescribedby="test-id" />,
+    );
+    expect(tree.find('NumberInput').props().ariaDescribedby).to.equal(
+      'test-id',
+    );
+    tree.find('Select').forEach(select => {
+      expect(select.props().ariaDescribedby).to.equal('test-id');
+    });
+    tree.unmount();
+  });
+  it('should pass aXe check when multiple aria-describedby ids are set', () => {
+    const date = {
+      day: makeField(''),
+      month: makeField(''),
+      year: makeField('1890'),
+    };
+    date.year.dirty = true;
+    date.month.dirty = true;
+    date.day.dirty = true;
```

Comment:
I know that this is following the existing pattern of other tests, but you may be happy to know that we are going to be improving the form component API and getting rid of this awkwardness.